src/ILCompiler.ReadyToRun/src/ObjectWriter/R2RPEBuilder.cs

```diff
@@ -596,27 +522,16 @@ protected override BlobBuilder SerializeSection(string name, SectionLocation loc
                         // the blob data goes out of sync and WriteContentTo outputs garbage.
                         sectionDataBuilder = PEResourceHelper.Relocate(inputSectionReader, rvaDelta);
                     }
+                    else if (name == ".text")
```

Comment:
Nit: There's a `TextSectionName` const up at the beginning of the file. But the use of the consts is not very consistent anyway...